packages/web/docs/src/components/ecosystem-management.tsx

```diff
@@ -132,7 +132,7 @@ function Illustration(props: { className?: string }) {
         </Edge>
         <div>
           <Node
-            title="Mesh"
+            title="Hive"
```

Comment:
I kinda feel like two tiles with the title `Hive` will seem like bug to the reader.

```suggestion
            title="Hive Gateway"
```

Would this make sense too?